fabs/README.txt

```diff
@@ -0,0 +1,96 @@
+# Floating Action Button components
+
+## FAB design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:fabs:<versions>")
+```
+
+### Primary
+
+**FAB**
+
+```kotlin
+object VitaminFabs {
+    @Composable
+    fun Primary(
+        icon: Painter,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: FabsColor = VitaminFabsColors.primary(),
+        sizes: FabSizes = VitaminFabSizes.default(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
+        onClick: () -> Unit
+    )
+}
+```
+
+Use FABs for primary, positive actions.
+
+The minimal usage of the component is the fab with an icon.
+
+```kotlin
+VitaminFabs.Primary(
+    icon = rememberVectorPainter(image = Icons.Filled.Add)
```

Comment:
Are you using icons from Material Extended Icons? You should show an example with our own icons, `ic_vtmn_*`.